docs/content/1.getting-started/2.basic-usage.md

```diff
@@ -8,34 +8,37 @@ The basics to get started with the Nuxt i18n module is to translate with Vue I18
 
 The basic to get started with **Nuxt i18n module** is to **translate with Vue I18n via the `vueI18n` option**
 
-So, let's get started configuring the following `nuxt.config`:
+So, let's get started by adding the module to `nuxt.config`:
 
 ```ts {}[nuxt.config.ts]
 export default defineNuxtConfig({
   modules: [
     '@nuxtjs/i18n'
   ],
   i18n: {
-    // add `vueI18n` option to `@nuxtjs/i18n` module options
-    vueI18n: {
-      legacy: false,
-      locale: 'en',
-      messages: {
-        en: {
-          welcome: 'Welcome'
-        },
-        fr: {
-          welcome: 'Bienvenue'
-        }
-      }
+    vueI18n: { configPath: './i18n.config.ts' } // if you are using custom path
+  }
+})
+```
+
+```ts {}[i18n.config.ts]
+export default defineI18nConfig({
+  legacy: false,
+  locale: 'en',
+  messages: {
+    en: {
+      welcome: 'Welcome'
+    },
+    fr: {
+      welcome: 'Bienvenue'
     }
   }
 })
 ```
 
-The `vueI18n` option is the same as `createI18n` function option of Vue I18n. `vueI18n` option is passed to the `createI18n` function via the nuxt plugin of this module internally.
+The `defineI18nConfig` function is the same as `createI18n` function option of Vue I18n. The configuration is passed to the `createI18n` function via the nuxt plugin (runtime) of this module internally.
```

Comment:
I think we should name `defineIVue18nConfig`, not `defineI18nConfig`, because that is coniguration for`vueI18n` option. And some people might misunderstand it as a nuxt i18n setting.